docs/app/pages/components/components-sections/input-controls/number-picker/snippets/app.ts

```diff
@@ -8,7 +8,6 @@ import { FormBuilder, Validators, FormGroup } from '@angular/forms';
 })
 export class AppComponent {
     form: FormGroup;
-
```

Comment:
Re-add this line break, most of our code follows the convention of having an empty line between properties and functions/constructor.

Reply:
Done the changes